packages/nextra-theme-docs/src/utils/normalize-pages.ts

```diff
@@ -14,7 +14,9 @@ function extendMeta(
   return Object.assign({}, fallback, meta, { theme })
 }
 
-export interface Item extends MdxFile {
+type FolderWithoutChildren = Omit<Folder, 'children'>
+
+export type Item = (MdxFile | FolderWithoutChildren) & {
```

Comment:
incorrect types, `item`, `docsItem`, `pageItem` and `menuItem` could be `Folder` but without children prop

proofs 👇

![image](https://user-images.githubusercontent.com/7361780/213948222-4c9753b3-e0db-4945-ada5-2be057ba5cf5.png)
![image](https://user-images.githubusercontent.com/7361780/213948241-9e3e93b8-8a37-4165-9807-49ee0a72df1f.png)